aspnetcore/grpc/native-aot.md

```diff
@@ -0,0 +1,56 @@
+---
+title: gRPC and native AOT
+author: jamesnk
+description: Learn about support for gRPC with .NET native ahead-of-time (AOT).
+monikerRange: '>= aspnetcore-8.0'
+ms.author: jamesnk
+ms.date: 04/06/2023
+uid: grpc/native-aot
+---
+# gRPC and native AOT
+
+By [James Newton-King](https://twitter.com/jamesnk)
+
+gRPC supports [.NET native ahead-of-time (AOT)](/dotnet/core/deploying/native-aot/) in .NET 8. Native AOT enables publishing gRPC client and server apps as small, fast native executables.
+
+> [!WARNING]
+> In .NET 8, not all ASP.NET Core features are compatible with native AOT. For more information, see [ASP.NET Core and native AOT compatibility](xref:fundamentals/native-aot#aspnet-core-and-native-aot-compatibility).
+
+## Getting started
+
+AOT compilation happens when the app is published. Native AOT is enabled with the `PublishAot` option.
+
+01. Add `<PublishAot>true</PublishAot>` to the gRPC client or server app's project file. This will enable native AOT compilation during publish and enable dynamic code usage analysis during build and editing.
```

Comment:
Replace numbers with all `1` and build system automatically increments.
Any reason you need 2 digits?